src/hooks/login/validateLoginForm.ts

```diff
@@ -2,7 +2,7 @@ import { ERROR } from '@/constants/message';
 import { isEmptyString } from '@/utils/validator';
 
 const isValidEmailFormat = (email: string): boolean => {
-  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  const emailRegex = /^[a-zA-Z0-9]+@[^\s@]+\.[^\s@]+$/;

```

Comment:
```suggestion
const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```
이런 정규식도 있는 거 같습니다! 추가로, 코드래빗과는 무관하게 클라이언트 환경에서는 엄격한 유효성 체크 보다는 `양식에 맞는 유효성` 처리를 해주는 것이 더 적합하다고 생각합니다!